media/js/wiki-compat-tables.js

```diff
@@ -145,10 +152,9 @@
                 $openCell.addClass('active').attr('aria-expanded', true);
 
                 // add measured height to history and to the cell/row it is being displayed beneath (CSS handles transition)
+                displayDetect = $table.find('thead td:first').first().css('z-index');
 
-                var displayDetect = $('.bc-table thead td').css('z-index');
```

Comment:
I think this was the line that slowed stuff down; this grabs every `table` and every `thead td` cell;  I've changed it to get the first `thead td` in this current table.